applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -739,8 +751,10 @@ static void discover_sink_cb(struct bt_conn *conn, struct bt_codec *codec, struc
 	}
 
 	/* At this point the location/channel index of the headset is always known */
-	memcpy(headsets[channel_index].sink_codec_cap, temp_codec_cap[temp_cap_index].cap,
-	       sizeof(temp_codec_cap[temp_cap_index].cap));
+	for (int i = 0; i < CONFIG_CODEC_CAPAB_COUNT_MAX; i++) {
```

Comment:
Hmm, why are these changes here in this PR?
This is changes from another sdk-nrf PR/commit

Reply:
Nevermind, this should be correct :)